src/views/assets/Cloud/Account/AccountList.vue

```diff
@@ -89,6 +93,7 @@ export default {
               name: 'LAN',
               title: this.$t('LAN'),
               icon: 'computer',
+              has: () => this.$route.query.category === 'host',
               callback: () => {
                 const providers = [lan]
                 this.providerConfig.providers = providers.map(
```

Comment:
There have been no identified irregularities, potential issues, or optimizations suggested in the provided HTML template code snippet from 2021 to the moment (January 13th, 2025).